packages/tools/pie-icons-configs/package.json

```diff
@@ -7,7 +7,8 @@
   "exports": {
     ".": "./configs.js",
     "./configs-react": "./configs-react.js",
-    "./configs-vue": "./configs-vue.js"
+    "./configs-vue": "./configs-vue.js",
+    "./configs": "./configs.js"
```

Comment:
Do we need to export this twice? I assume `./configs.js` is already getting exported on line 8?

Reply:
resolved this [here](https://github.com/justeattakeaway/pie/pull/585/files#diff-3fa9cef765d67886e2775ac19cd61a60dcf31185b36c776537bb43292344f565R8) - I've renamed the export from `.` to the filename so it's clearer for the import 👍 Closing this PR as lots of changes had gone through and it was less confusing to start from most recent main 😄